ledger/complete/wal/compactor.go

```diff
@@ -76,7 +80,9 @@ func (c *Compactor) Run() error {
 	c.Lock()
 	defer c.Unlock()
 
-	err := c.createCheckpoints()
+	latestCheckPoint := -1
```

Comment:
This is a local variable, so it will be -1 everytime `Run` is called and will always emit the `onNext()`, or do I miss something?

Reply:
🤦 
Really left my brain at the cleaner's today.